client/cli/src/client_proxy.rs

```diff
@@ -518,6 +519,7 @@ impl ClientProxy {
             sender_sequence_number,
             max_gas_amount.unwrap_or(MAX_GAS_AMOUNT),
             gas_unit_price.unwrap_or(GAS_UNIT_PRICE),
+            TypeTag::Struct(lbr_struct_tag()),
```

Comment:
Overall this PR looks good to me.

Could you however pull this into its own function in account config? e.g `lbr_type_tag()` so that this doesn't have to be explicitly constructed everywhere?